src/main/java/com/traveltime/sdk/dto/responses/errors/IOError.java

```diff
@@ -1,8 +1,10 @@
 package com.traveltime.sdk.dto.responses.errors;
 
 import lombok.AllArgsConstructor;
+import lombok.Getter;
 import lombok.NonNull;
 
+@Getter
```

Comment:
I think here you also want `getMessage` to return `errorMessage`?

Reply:
what do you mean?

Reply:
getter will generate getErrorMessage

Reply:
Yes, but I think the `getMessage` method that now returns `cause.getMessage` should actually return `errorMessage`. That way if you call `getMessage` on any error returned by the sdk you will get a message provided by us.
Currently `getMessage` will return a message from some exception from IOError and custom messages everywhere else.

Reply:
Okey, I changed it